File: Python/1957-delete-characters-to-make-fancy-string.py
class Solution:
    def makeFancyString(self, s: str) -> str:
        output = []

        for ch in s :
            if len(output) < 2 or output[-2] != output[-1] or output[-1] != ch :
                output.append(ch)

        return "".join(output)
        

File: Javascript/1957-delete-characters-to-make-fancy-string.js
/**
 * @param {string} s
 * @return {string}
 */
var makeFancyString = function(s) {
    const arr = [];
    for (let i in s) {
        if (arr.length < 2 || arr[arr.length - 2] != arr[arr.length - 1] || arr[arr.length - 1] != s[i]) arr.push(s[i]);
    }
    return arr.join("");
};